Formula/flake8.rb

```diff
@@ -30,15 +30,33 @@ class Flake8 < Formula
     sha256 "379358498f58f69157b53f59f46aefda0e9a3eb81365238f69fbedf7014e21ab"
   end
 
+  resource "configparser" do
+    url "https://files.pythonhosted.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz"
+    sha256 "5308b47021bc2340965c371f0f058cc6971a04502638d4244225c49d80db273a"
+  end
+
+  resource "enum34" do
+    url "https://files.pythonhosted.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz"
+    sha256 "8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1"
+  end
+
   def install
     ENV.prepend_create_path "PYTHONPATH", "#{libexec}/vendor/lib/python2.7/site-packages"
 
-    %w[pycodestyle pyflakes mccabe].each do |dependency|
+    %w[pycodestyle pyflakes mccabe enum34].each do |dependency|
       resource(dependency).stage do
         system "python", *Language::Python.setup_install_args(libexec/"vendor")
       end
     end
 
+    resource("configparser").stage do
+      # NOTE(sigmavirus24): configparser uses namespace packages which do not
+      # play well with --single-version-externally-managed
```

Comment:
I actually don't know what easy_install is doing that allows this to work but it's probably bad. :D The way namespace packages are handled with --single-version-externally-managed is to write a special .pth file into site-packages, which unfortunately doesn't help when the installation prefix isn't a blessed sitedir from which .pth files are read. The workaround we've had is to touch an __init__.py in the namespace directory.

But! It's probably better to convert flake8 to use the new virtualenv encapsulation where namespace packages just work; basically you should add `include Language::Python::Virtualenv` at the very top of the class definition and replace the install method with

```ruby
def install
  virtualenv_install_with_resources
end
```